multi-node/vagrant/turbo-server-config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "turbo-server-address": "1.2.3.4",
```

Comment:
Can we use the similar kubeturbo config file as we used in other installation, instead of having a file with different keyname and serve the same purpose?